tests/providers/microsoft/azure/transfers/test_local_to_adls.py

```diff
@@ -59,22 +56,3 @@ def test_execute_raises_for_bad_glob_val(self, mock_hook):
         with pytest.raises(AirflowException) as ctx:
             operator.execute(None)
         assert str(ctx.value) == "Recursive glob patterns using `**` are not supported"
-
-    @mock.patch("airflow.providers.microsoft.azure.transfers.local_to_adls.AzureDataLakeHook")
-    def test_extra_options_is_passed(self, mock_hook):
-        operator = LocalFilesystemToADLSOperator(
-            task_id=TASK_ID,
-            local_path=LOCAL_PATH,
-            remote_path=REMOTE_PATH,
-            extra_upload_options={"run": False},
-        )
-        operator.execute(None)
-        mock_hook.return_value.upload_file.assert_called_once_with(
-            local_path=LOCAL_PATH,
-            remote_path=REMOTE_PATH,
-            nthreads=64,
-            overwrite=True,
-            buffersize=4194304,
-            blocksize=4194304,
-            run=False,  # extra upload options
-        )
```

Comment:
why do we remove this test if no changes were made in the `LocalFilesystemToADLSOperator` itself?

Reply:
Oeps that will be a wrong refactoring of my part :)

Reply:
I still have following error related to docs but dunno what I did wrong, as I only moved docs from local_to_adls transfers to adls in operators.  I always receive following error in static checks and when I compare the files I edited I don't see any issues:

```
apache-airflow-providers-microsoft-azure                     Traceback (most recent call last):
  apache-airflow-providers-microsoft-azure                       File "/usr/local/lib/python3.8/site-packages/sphinx/cmd/build.py", line 281, in build_main
  apache-airflow-providers-microsoft-azure                         app.build(args.force_all, args.filenames)
  apache-airflow-providers-microsoft-azure                       File "/usr/local/lib/python3.8/site-packages/sphinx/application.py", line 347, in build
  apache-airflow-providers-microsoft-azure                         self.builder.build_update()
  apache-airflow-providers-microsoft-azure                       File "/usr/local/lib/python3.8/site-packages/sphinx/builders/__init__.py", line 307, in build_update
  apache-airflow-providers-microsoft-azure                         self.build(['__all__'], to_build)
  apache-airflow-providers-microsoft-azure                       File "/usr/local/lib/python3.8/site-packages/sphinx/builders/__init__.py", line 326, in build
  apache-airflow-providers-microsoft-azure                         updated_docnames = set(self.read())
  apache-airflow-providers-microsoft-azure                       File "/usr/local/lib/python3.8/contextlib.py", line 120, in __exit__
  apache-airflow-providers-microsoft-azure                         next(self.gen)
  apache-airflow-providers-microsoft-azure                       File "/usr/local/lib/python3.8/site-packages/sphinx/util/logging.py", line 218, in pending_warnings
  apache-airflow-providers-microsoft-azure                         memhandler.flushTo(logger)
  apache-airflow-providers-microsoft-azure                       File "/usr/local/lib/python3.8/site-packages/sphinx/util/logging.py", line 183, in flushTo
  apache-airflow-providers-microsoft-azure                         logger.handle(record)
  apache-airflow-providers-microsoft-azure                       File "/usr/local/lib/python3.8/logging/__init__.py", line 1599, in handle
  apache-airflow-providers-microsoft-azure                         self.callHandlers(record)
  apache-airflow-providers-microsoft-azure                       File "/usr/local/lib/python3.8/logging/__init__.py", line 1661, in callHandlers
  apache-airflow-providers-microsoft-azure                         hdlr.handle(record)
  apache-airflow-providers-microsoft-azure                       File "/usr/local/lib/python3.8/logging/__init__.py", line 950, in handle
  apache-airflow-providers-microsoft-azure                         rv = self.filter(record)
  apache-airflow-providers-microsoft-azure                       File "/usr/local/lib/python3.8/logging/__init__.py", line 811, in filter
  apache-airflow-providers-microsoft-azure                         result = f.filter(record)
  apache-airflow-providers-microsoft-azure                       File "/usr/local/lib/python3.8/site-packages/sphinx/util/logging.py", line 426, in filter
  apache-airflow-providers-microsoft-azure                         raise exc
  apache-airflow-providers-microsoft-azure                     sphinx.errors.SphinxWarning: 
  /opt/airflow/docs/apache-airflow-providers-microsoft-azure/_api/airflow/providers/microsoft/azure/operators/adls/index.rst:17:Definition list ends without a blank line; unexpected 
  unindent.
  apache-airflow-providers-microsoft-azure                     
  apache-airflow-providers-microsoft-azure                     [91mWarning, treated as error:[39;49;00m
  apache-airflow-providers-microsoft-azure                     
  /opt/airflow/docs/apache-airflow-providers-microsoft-azure/_api/airflow/providers/microsoft/azure/operators/adls/index.rst:17:Definition list ends without a blank line; unexpected 
  unindent.
  apache-airflow-providers-microsoft-azure                    : ################################################################################
  
  
No packages to retry. No more passes are needed.
No packages to retry. No more passes are needed.
No packages to retry. No more passes are needed.
fThere were some build errors remaining.


############################## Start spelling errors summary ##############################

```